app/models/campaign.rb

```diff
@@ -0,0 +1,27 @@
+# This "model" is not backed by the database. Its main purpose is giving one of
+# our domain concepts an actual representation in code.
+class Campaign
```

Comment:
Given this is not an AR model, should we maybe put it in services? I'm fine with it here but wasn't sure if we wanted to keep these to AR models or not.

Reply:
I much prefer this class being in model as it's clearly solving the problem of missing domain object. There's also prior art for non-AR backed classes in `/models` in `language.rb`

Reply:
Here's how I generally think about this:

* `app/model`: Nouns, e.g. `User`, `Article`. Whether or not a model is DB-backed doesn't matter for this.
* `app/services`: Verbs, e.g. `User::Delete`, `Article::BustCache` (made-up examples). Actions performed on those nouns.

We're currently not quite there, as our services often use noun-names too, but for me, this separation has worked well in the past. I also tend to namespace services by the object the operate on as in the examples given above.

But I'm fine with everything the team agrees on.